docs/source/reference/templates.md

```diff
@@ -1,18 +1,21 @@
 # Working with templates and UI
 
 The pages of the JupyterHub application are generated from
-[Jinja](http://jinja.pocoo.org/) templates. These allow the header, for
-example, to be defined once and incorporated into all pages. By providing
-your own templates, you can have complete control over JupyterHub's
+[Jinja](http://jinja.pocoo.org/) templates. This allows the header to be defined once and incorporated into all pages. By providing your templates, you can have complete control over JupyterHub's
 appearance.
+This document will provide information on how to:
+
+- [add custom templates](#custom-templates)
```

Comment:
Let's not add in-body table-of-contents links. We already have these links that will be auto-generated and correct in the sidebar of the docs, whereas these links will break without warning if the spelling of a heading is changed.

Reply:
Okay, I have noted the reviews and will implement them. 

Thank you